src/devana/utility/init_params.py

```diff
@@ -0,0 +1,44 @@
+from typing import Callable, Any, Optional, Set
+from inspect import signature, BoundArguments
+from functools import wraps
+
+
+def init_params(skip: Optional[Set[str]] = None) -> Callable:
+    """A decorator that assigns method parameters as instance attributes.
+    The attribute must have a setter or exist as an instance or class variable."""
```

Comment:
Please document the use of this in a second paragraph and how it is supposed to work technically.